gc/base/CollectorLanguageInterface.cpp

```diff
@@ -31,28 +31,33 @@ struct OMR_VMThread;
 void
 MM_CollectorLanguageInterface::writeBarrierStore(OMR_VMThread *omrThread, omrobjectptr_t parentObject, fomrobject_t *parentSlot, omrobjectptr_t childObject)
 {
-	GC_SlotObject slotObject(omrThread->_vm, parentSlot);
-	slotObject.writeReferenceToSlot(childObject);

```

Comment:
I just ran into this problem yesterday.  I think a better solution would be to just pass the OMR_VMThread through to generationalWriteBarrierStore and concurrentWriteBarrierStore as the first parameter and fetch the MM_EnvironmentStandard inside of those calls.  I think it cleans up the API and makes it easier for them to be called from other locations.